svelte-kit-scss/src/lib/components/Profile/OrganizationProfile/OrganizationDetails/OrganizationMembers.spec.ts

```diff
@@ -0,0 +1,31 @@
+import { it, expect } from 'vitest';
+import { render } from '@testing-library/svelte';
+import OrganizationMembers from './OrganizationMembers.svelte';
+import { MOCK_GITHUB_ORGANIZATION_MEMBERS } from '$lib/helpers/mocks/users';
+
+const members = MOCK_GITHUB_ORGANIZATION_MEMBERS;
+
+describe('OrganizationMembers', () => {
+  function organizationMembersTree() {
+    return render(OrganizationMembers, {
+      members,
+    });
+  }
+
+  beforeEach(() => {
+    organizationMembersTree();
+  });
```

Comment:
```suggestion
  beforeAll(organizationMembersTree);
```

We can use `beforeAll` here to prevent unnecessary rerendering.

Reply:
I couldnt get beforeAll working. I get an error on the second item in the array.

Reply:
My bad, typically beforeAll would work when you want to set something up that is consistent throughout all tests. But the "screen" resets after each test, so we need to rerender.